### ISSUE TITLE
Connection Service - Not Used

### ISSUE THREAD

**Issue by mohit67890** [AgoraIO-Usecase/Video-Calling#27]
Hi Team,

Thank You for the sample application..
I was checking the entire code of this App, I found the implementation of the ConnectionService in Android. 
However, as per my inspection the connection service is not being used in the app, wherein it say that is not supported. 

I wanted to confirm if I am right that the Connection Service is not used in the app as of now and RTM is being used for call invitations?
Also, I wanted to know why it is now supported yet. 

Thanks,
Mohit

**Comment by plutoless** [comment]
yes RTM is being used for call invitations. We use connection service only for call interface. If you want to support push you will need to implement corresponding push notification service.
